src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -8,6 +8,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <NoWarn>1591</NoWarn>
     <CLSCompliant>false</CLSCompliant>
+    <IsTargetingNetCoreApp Condition="'$(TargetGroup)'=='netcoreapp' OR '$(TargetGroup)'=='uap' OR '$(TargetGroup)'=='uapaot' OR '$(TargetGroup)'=='netcoreappaot'">true</IsTargetingNetCoreApp>
```

Comment:
There is existing centrally defined `TargetsNetCoreApp` that is defined for netcoreapp only. This one is very similar, but different.

I think it would be better to just use `$(TargetsNetCoreApp)'=='true' OR '$(TargetsUap)' == 'true'` condition here. It is used in number of other places for situations like this.

Reply:
I just copied what use in S.N.Vectors that you said was an okay pattern back in March: https://github.com/dotnet/corefx/pull/27531#discussion_r171911909

Reply:
We did not have `TargetsNetCoreApp` back in March